Projekty/Projekt2/Gruszkowska_Gryszkiewicz_Chylak/KM1.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p><strong>Bardzo dobrze wykonany KM1</strong> (szkoda, że mało opisów zastosowanych metod i wniosków z analiz)</p><ul><li>Bardzo trafne wnioski - super, że następnym planowanym krokiem jest lematyzacja</li><li><strong>Bardzo doceniam zastosowanie technik NLP, które nie są omawiane na tym przedmiocie</strong></li></ul><p> </p><p>Sugestie w kontekście dalszych prac:</p><p>Można jeszcze pomyśleć o innych metodach analizy słów z użyciem pakietu NLTK:</p><ul><li>być może słowa, które mają w sobie myślnik warto rozdzielić na dwa słowa,</li><li>może są wyrazy bliskoznaczne (wówczas można byłoby się zastanowić nad unifikacją, chyba że „wybór jednego&nbsp;możliwych słów do wyrażenia treści” determinuje przynależność do danej księgi)&nbsp;- niemniej jednak analiza pod kątem podobieństwa słów może być czasochłonna</li><li>itp.</li></ul><p>Powyższe sugestie nie są zobowiązujące, ponieważ nie jest to przedmiot z obszaru NLP stricte. Niemniej jednak może to Państwa zainteresować.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>